locale/en/docs/guides/event-loop-timers-and-nexttick.md

```diff
@@ -76,12 +76,14 @@ actually uses - are those above._
 
 * **timers**: this phase executes callbacks scheduled by `setTimeout()`
  and `setInterval()`.
-* **I/O callbacks**: executes almost all callbacks with the exception of
- close callbacks, the ones scheduled by timers, and `setImmediate()`.
+* **pending callbacks**: executes I/O callbacks deferred to the next loop
```

Comment:
IMO, `I/O` is more appropriate than `pending` here, 'cause `close` callbacks wouldn't be executed here, so `I/O` makes more sense in this phase.

Reply:
@marswong Please keep in mind that I'm not a C programmer, so most of my claims are based on [libuv's event loop overview](http://docs.libuv.org/en/v1.x/design.html). Perhaps you could point to source code where Node.js has deviated or where the libuv doc is off.

`I/O` alone seems to indicate this is where all or most I/O callbacks are executed. However, this is the responsibility of the `poll` phase. Only some callbacks that were deliberately delayed will be executed in this phase (hence "pending").

Originally, I suggested renaming this phase to `pending I/O callbacks`, but that seemed a little long (especially for the ascii art), so I shortened it to `pending callbacks` and used the brief description to elaborate. As a reference, libuv calls this phase `call pending callbacks`.

Reply:
good point :)

Reply:
More clear, thx.